Makefile

```diff
@@ -93,17 +93,21 @@ endif
 docker-tests:
 	export OPENQA_LOGFILE=/opt/openqa/openqa-debug.log ;\
 	if test "x$$FULLSTACK" = x1 || test "x$$SCHEDULER_FULLSTACK" = x1 || test "x$$DEVELOPER_FULLSTACK" = x1; then \
-		git clone https://github.com/os-autoinst/os-autoinst.git ../os-autoinst ;\
-		cd ../os-autoinst ;\
-		cpanm -n --mirror http://no.where/ --installdeps . ;\
-		if [ $$? -eq 0 ]; then\
-			sh autogen.sh && make ;\
-			cd - ;\
-			eval $$(dbus-launch --sh-syntax) ;\
-			export PERL5OPT="$$PERL5OPT $$HARNESS_PERL_SWITCHES" ;\
-		else \
-			echo "OS autoinst dependencies not match. Please check output above" ;\
-			exit 1 ;\
+		if test -z "$$CUSTOM_OS_AUTOINST"; then\
+			git clone https://github.com/os-autoinst/os-autoinst.git ../os-autoinst ;\
+			cd ../os-autoinst ;\
+			cpanm -n --mirror http://no.where/ --installdeps . ;\
+			if [ $$? -eq 0 ]; then\
+				sh autogen.sh && make ;\
+				cd - ;\
+				eval $$(dbus-launch --sh-syntax) ;\
+				export PERL5OPT="$$PERL5OPT $$HARNESS_PERL_SWITCHES" ;\
```

Comment:
I only see whitespace... oh wait, I need to scroll to the right ;) IMHO complexity /indentation too high

Reply:
This should likely be a separate script. And since this sprint is about reducing technical debt I'll take the time to change it.